rosdep/python.yaml

```diff
@@ -6390,6 +6390,12 @@ python3-future:
   openembedded: [python3-future@meta-python]
   rhel: ['python%{python3_pkgversion}-future']
   ubuntu: [python3-future]
+python3-gdal:
+  debian: [python3-gdal]
```

Comment:
please add the arch package https://archlinux.org/packages/community/x86_64/python-gdal/